python/dgl/nn/pytorch/hetero.py

```diff
@@ -169,7 +169,7 @@ def forward(self, g, inputs, mod_args=None, mod_kwargs=None):
                     continue
                 dstdata = self.mods[etype](
                     rel_graph,
-                    inputs[stype],
+                    (inputs[stype], inputs[dtype]),
```

Comment:
Test failed. If we want to add this change, all dst node feature should be there in the dict when calling forward. This might be a breaking change, we may also need to test on heterograph examples. If we don't want to break existing code, we may instead using an option, when user use graphSAGE, the error message will inform the option. If we think it's not a huge break on existing code, we can go on with this change and just fix test.